structure/0007-community-team.md

```diff
@@ -256,6 +249,28 @@ The vote has passed when one of the following is true, but only if no failing co
 As detailed above, there should be no situations where a vote ends in a tie. If a situation arises that the voting
 process is unable to cover, the vote will fail and will need to be re-tried when the process has been amended.
 
+### Determining voting eligibility
+
+#### Community Team
+
+All Community Team members are eligible to vote in polls, assuming they have been a member of the Community Team for
+at least two weeks and are not a Trainee Community Moderator. Trainees, as well as Community Team members who have not
+held their role for at least two weeks, are not voting members and should not be considered for internal votes. This is
+to ensure that all voters have had some time to get familiar with the community and how it's run, but non-voting
+members of the Community Team are always welcome to express their opinions and provide insight if they wish to.
+
+Additionally, any Community Team member who has not interacted in any Quilt community space for more than two weeks,
+internal or public, is considered inactive. Inactive members are not counted as a positive vote, negative vote, or
+abstention. These members are still eligible to vote in polls, but by voting, they are no longer considered inactive.
```

Comment:
This doesn't state that inactive members aren't included in tallies either, should put that in for consistency.